assets/src/edit-story/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -135,26 +137,29 @@ const getPlaceholderResource = (file) => {
  * Generates a resource object from a local File object.
  *
  * @param {File} file File object.
- * @return {Promise<import('./createResource').Resource>} Resource object.
+ * @return {Object} Object containing resource object and poster file.
```

Comment:
This should probably be `Promise<Object<{resource: import('./createResource').Resource>, posterFile: File}>>`

Reply:
Sadly, the above doesn't work. I spent a while to get something to get a return that looks right but doesn't make a link error.

Reply:
See https://github.com/google/web-stories-wp/pull/7317/commits/a5360e9c7a5a53208bdc6289147ea4d4d446352f